libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -626,7 +646,7 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
                     ObjectPath.SetPathValue(stepContext.ActiveDialog.State, Options, dialogOptions);
 
                     // Get multi-turn prompts card activity.
-                    var message = QnACardBuilder.GetQnAPromptsCard(answer, dialogOptions.ResponseOptions.CardNoMatchText);
+                    var message = QnACardBuilder.GetQnADefaultResponse(answer, dialogOptions.ResponseOptions.DisplayPreciseAnswerOnly);
                     await stepContext.Context.SendActivityAsync(message).ConfigureAwait(false);
```

Comment:
>wait stepContext.Context.SendActivityAsync(message).ConfigureAwait(false) [](start = 21, length = 73)

Send the cancellation token here.

Reply:
implemented the suggestion

Reply:
Thanks. Shall fix it.

---
In reply to: [433510960](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433510960) [](ancestors = 433510960)